### DISCUSSION TITLE
How to enable /f/b -> /foo/bar dir path completion

### DISCUSSION THREAD

**Discussion by nimish** [marlonrichert/zsh-autocomplete#201] [Q&A] [answered]
How would I configure this? It's something to do with the `expand` completer, but I can't figure out the right style

**Comment by marlonrichert** [comment, marked as answer]
```zsh
zstyle ':completion:*:paths' path-completion yes
```

This has nothing to do with the `_expand` completer, though. That one takes care only of substituting [expansions](http://zsh.sourceforge.net/Doc/Release/Expansion.html).

**Reply by nimish** [reply]
Oh weird, in the `compinstall` source it seemed to involve an `expand` option. Thanks!
